docs/home-page-refs.rst

```diff
@@ -0,0 +1,15 @@
+Ubuntu Pro Client references
+============================
+
+Our Reference section is for quickly checking what commands are available to you with the Ubuntu Pro Client, and how to interact with them.  
```

Comment:
Right now, our references section is more aimed at delivering support information for the pro client. For example, stating about network requirements, support matrix for some services, api defintions and things like that, so it is not command based per se.

But let us know if this is the right section for such information

Reply:
It sounds like a good place for it! I'll update the text accordingly